### PR TITLE
Update docs for intpoly module

### PR DESCRIPTION
**Pull request by ArifAhmed1995**
Adds a section for the newly created `intpoly` module.

### CODE REVIEW

**Comment by ArifAhmed1995** [comment]
@certik Can you review please ? Should I add more examples ?

**Comment by certik** [comment]
@ArifAhmed1995 thank you for writing it up. I would also add there something along the lines of:
```
Uses evaluation techniques as described in Chin et al(2015)[1]
References
===========
[1] : http://dilbert.engr.ucdavis.edu/~suku/quadrature/cls-integration.pdf
```
so that people know where to get more information. However, please fix the citation at both places to:

Chin, Eric B., Jean B. Lasserre, and N. Sukumar. "Numerical integration of homogeneous functions on convex and nonconvex polygons and polyhedra." Computational Mechanics 56.6 (2015): 967-981.

As that is the correct way to cite the paper. The pdf location might change.

**Comment by ArifAhmed1995** [comment]
@certik Is it fine now ?

File: doc/src/modules/integrals/integrals.rst [hunk 1]
```diff
@@ -133,6 +133,8 @@ Integration over Polytopes
 .. module:: sympy.integrals.intpoly
 
 The ``intpoly`` module in SymPy implements methods to calculate the integral of a polynomial over 2/3-Polytopes.
+Uses evaluation techniques as described in Chin et al(2015)[1]
```
Comment (certik) [inline doc/src/modules/integrals/integrals.rst]: `Chin et al. (2015) [1].`. Notice the dot after `al.`.

File: sympy/integrals/intpoly.py [hunk 2]
```diff
@@ -6,7 +6,11 @@
```
Comment (certik) [inline sympy/integrals/intpoly.py]: Chin et al. (2015) [1].

**Review by certik — CHANGES_REQUESTED** [review]
Thanks. Few more minor comments.

**Comment by certik** [comment]
You have to put the dot after `al.`, because `et al.` is an abbreviation from Latin `et alii`, meaning "and others".

**Comment by ArifAhmed1995** [comment]
Is it fine now ? I just updated the PR.

**Review by certik — APPROVED** [review]

**Comment by certik** [comment]
Yes, that looks good now. +1 to merge. Please ping me once tests pass, so that I can merge.

**Comment by ArifAhmed1995** [comment]
@certik I don't why that test fails on Travis. It runs fine locally. Maybe a restart would help.
This is the job : https://travis-ci.org/sympy/sympy/jobs/272747201

**Comment by certik** [comment]
I restarted the job. It seems unrelated to this PR.

**Comment by ArifAhmed1995** [comment]
@certik Did the test fail again as well or is the restart in queue and hasn't happened yet ?

**Comment by certik** [comment]
@ArifAhmed1995 it's caused by #13270. Let me see how to fix it.

**Comment by certik** [comment]
I merged a fix into master, and rebased your PR on top of the latest master. All tests should now pass. @ArifAhmed1995 please ping me once they do, so that I can merge this.

**Comment by ArifAhmed1995** [comment]
@certik The tests finally pass now...

**Comment by certik** [comment]
Perfect, thanks!

**Comment by ArifAhmed1995** [comment]
@certik I can't see this new section on the SymPy docs page. Will it be there once the next release is done ?

**Comment by gxyd** [comment]
They are present in `dev` not in `latest`, here is the link http://docs.sympy.org/dev/modules/integrals/integrals.html . Also I think Aaron does the re-building of the docs manually himself after a few days may be.

**Comment by ArifAhmed1995** [comment]
@gxyd Oh I see. Thank you for explaining !

**Pull request merged**